packages/ui-extensions/src/surfaces/customer-account/targets.ts

```diff
@@ -89,24 +89,24 @@ export interface CustomerAccountExtensionTargets {
     StandardApi<'customer-account.order-index.block.render'>,
     AllComponents
   >;
-  'customer-account.account-information.block.render': RenderExtension<
-    StandardApi<'customer-account.account-information.block.render'>,
+  'customer-account.profile.block.render': RenderExtension<
+    StandardApi<'customer-account.profile.block.render'>,
```

Comment:
These targets name modification are from this latest pr:
https://github.com/Shopify/ui-extensions/pull/1288/files